tests/PHPUnit/Integration/Plugin/ManagerTest.php

```diff
@@ -147,8 +147,8 @@ private function getCacheForTrackerPlugins()
 
     private function assertOnlyTrackerPluginsAreLoaded($expectedPluginNamesLoaded)
     {
-        // should currently load between 10 and 27 plugins
-        $this->assertLessThan(27, count($this->manager->getLoadedPlugins()));
+        // should currently load between 10 and 28 plugins
+        $this->assertLessThan(28, count($this->manager->getLoadedPlugins()));
```

Comment:
Took me a while to figure out why this increased but that's because Marketplace is now a tracker plugin